src/__tests__/calculate.test.js

```diff
@@ -0,0 +1,46 @@
+// eslint-disable-next-line import/extensions
+import calculate from '../logic/calculate';
+// import operate from '../logic/operate';
```

Comment:
- [ ] Please remove eslint disablement line and try to fix linters errors yourself. 
[hint:] I removed this line and there is no error found.

- [ ] Please remove any commented(unused) code like line 3 to make your code clean.